custom_components/luxtronik/sensor.py

```diff
@@ -5,7 +5,7 @@
 import voluptuous as vol
 from homeassistant.components.sensor import PLATFORM_SCHEMA
 from homeassistant.const import (CONF_FRIENDLY_NAME, CONF_ICON, CONF_ID,
-                                 CONF_SENSORS)
+                                 CONF_SENSORS, CONF_STATE_CLASS)
```

Comment:
`homeassistant.const` has no `CONF_STATE_CLASS`, import that from `.const` instead

Add `CONF_STATE_CLASS = "state_class"` to `const.py`